spec/javascript/test/containers/AirportShowContainer.js

```diff
@@ -0,0 +1,34 @@
+import '../../testHelper.js'
+
+import AirportShowContainer from '../../../../app/javascript/react/containers/AirportShowContainer'
+import AirportShowTile from '../../../../app/javascript/react/components/AirportShowTile'
+
+describe('AirportShowContainer', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    jasmineEnzyme();
+    wrapper = shallow(
+      <AirportShowContainer />
+    )
+  })
+
+  it('should should have the specified initial state', () => {
+    expect(wrapper.state()).toEqual({ airport: {id: '', name: '', airport_code: '', location: '', description: ''} });
+  });
+
+  it('renders an h1 tag titled Airport', () => {
+    expect(wrapper.find('h1')).toBePresent()
+    expect(wrapper.find('h1').text()).toContain("Airport")
+  });
+
+  it('should render Airport Tile Components with specific props', () => {
+    wrapper.setState({ airport: {id: 1, name: "Logan", location: "Boston", airport_code: "BOS", description: "Anything"} });
```

Comment:
This is a wonderful start! However, you need to implement `fetchMock` instead of manually setting the state of your wrapper in your test. This way, you know that your fetch is setting the state as you'd hoped 🙂

https://learn.launchacademy.com/teams/boston-24/curricula/on-campus-boston-24/lesson_groups/week_4:_advanced_react/lessons/faking-fetch-with-fetch-mock